exchange/utils.go

```diff
@@ -177,48 +172,79 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 			}
 		}
 
+		ipConf := privacy.IPConf{IPV6: auctionReq.Account.Privacy.IPv6Config, IPV4: auctionReq.Account.Privacy.IPv4Config}
+
+		reqWrapper := cloneBidderReq(bidderRequest.BidRequest)
```

Comment:
This seems like a good compromise. As a follow-up, we'll want to pass the request wrapper all the way down, but that's not possible right now since we lose it in `getAuctionBidderRequests` and that's too much work for this PR.

Reply:
Right. Better to refactor it separately.